src/main/java/org/assertj/core/api/AbstractFloatArrayAssert.java

```diff
@@ -223,6 +225,32 @@ public SELF contains(float... values) {
     return myself;
   }
 
+  /**
+   * Verifies that the actual array contains the values of the given array, in any order.
+   * <p>
+   * Example:
+   * <pre><code class='java'> // assertion will pass
+   * assertThat(new float[] { 1.0f, 2.0f }).contains(new Float[] { 1.0f, 2.0f });
+   * assertThat(new float[] { 1.0f, 2.0f }).contains(new Float[] { 1.0f, 2.0f });
+   * assertThat(new float[] { 1.0f, 2.0f }).contains(new Float[] { 1.0f });
+   *
+   * // assertion will fail
+   * assertThat(new float[] { 1.0f, 2.0f }).contains(new Float[] { 3.0f });</code></pre>
+   *
+   * @param values the given values.
+   * @return {@code this} assertion object.
+   * @throws NullPointerException if the given argument is {@code null}.
+   * @throws IllegalArgumentException if the given argument is an empty array.
+   * @throws AssertionError if the actual array is {@code null}.
+   * @throws AssertionError if the actual array does not contain the given values.
+   * @since 3.19.0
+   */
+  public SELF contains(Float[] values) {
```

Comment:
Please add the corresponding variants for all the `containsXXX(double[], Offset<Float>)` methods, e.g.:

```java
public SELF contains(Float[] values, Offset<Float> precision)
```

Reply:
https://github.com/assertj/assertj-core/blob/c0f48d3858e047ed0421a36c435e6f1b5503edcb/src/main/java/org/assertj/core/api/AbstractFloatArrayAssert.java#L248

It's already there. Also the rest of the methods you're talking about.